### ISSUE TITLE
SyntaxError: invalid syntax for L131 in names.py

### ISSUE THREAD

**Issue by simonharnqvist** [biocore-ntnu/pyranges#380]
Hi,

I think there might be a bug in version 0.1.0 (or at least an incompatibility with Python 3.11.4). PyRanges 0.0.129 is working well for me, but I get a `SyntaxError` with the new version.

Here's the relevant bit of traceback when I try to import PyRanges:
```
  File "[conda path]/pyranges/core/names.py", line 131
    class UnaryOperation[T: "RangeFrame"](Protocol):
                        ^
SyntaxError: invalid syntax
```

Thanks
Simon

**Comment by endrebak** [comment]
ACK.

**Comment by endrebak** [comment]
So the problem is that pip install pyranges installs v1?!

**Comment by endrebak** [comment]
I'll do it when I get home. It requires 2FA which I have at home. I'll get you a PyPI auth too, Marco

**Comment by endrebak** [comment]
Five hours

**Comment by marco-mariotti** [comment]
Thanks Endre. Yes somehow pyranges1 version 0.1.0 was uploaded to pip pyranges instead than to pyranges1.

I think this was a temporary problem (early attempts while you setup the github action?), since afterwards I've used github action to publish to Pypi version 1.0.0 and it went to the right place.

Yes please make me a maintainer of both

**Comment by endrebak** [comment]
Yes, we should also find out why this happened.

**Comment by endrebak** [comment]
@marco-mariotti One way you could have solved it is to bump the version and then run the build and upload wheels pipeline. Just saying for the next time.

**Comment by endrebak** [comment]
I'll try to make you a maintainer for both on PyPI too.

**Comment by endrebak** [comment]
Fixed in 0.1.1. Please try downloading the latest and tell us if it worked or not.

Thanks for reporting :)
